package.json

```diff
@@ -29,7 +29,9 @@
     "jest-environment-jsdom": "29.7.0",
     "prettier": "3.0.3",
     "react": "18.2.0",
+    "react-aria-components": "1.0.0-beta.0",
     "react-dom": "18.2.0",
+    "tailwindcss-react-aria-components": "1.0.0-beta.0",
```

Comment:
we sadly have to add this in a separate PR because this causes issue on other components (e.g. button active state is no longer working)